.github/workflows/check-catalyst.yaml

```diff
@@ -384,7 +384,9 @@ jobs:
         # cuda-quantum is added manually here.
         # It can't be in requirements.txt as that will break
         # macOS requirements.txt
-        python3 -m pip install cuda-quantum
+        # TODO: Remove version condition when https://github.com/PennyLaneAI/catalyst/pull/617
+        # is merged.
+        python3 -m pip install 'cuda-quantum<0.7'
```

Comment:
This should be reverted I think since it was handled in a separate PR

Reply:
Done